dptb/tests/test_dist_data.py

```
@@ -0,0 +1,38 @@
+import pytest
+from dptb.data import AtomicData, _keys
+import ase.io as io
+try:
+    import mpi4py
+    _MPI = True
+except:
```

Ruff (0.14.10)

Do not use bare `except`

(E722)